### DISCUSSION TITLE
Not an issue

### DISCUSSION THREAD

**Discussion by aarondfrancis** [oneduo/nova-file-manager#3] [General]
Hey! This is Aaron from the Twitch stream yesterday. I took a look at the package and honestly it looks amazing. It seems to be really well architected and I dove into some of the classes and they look super clean. You should be proud of this! I don't use Nova on any projects, but if I do I'll absolutely reach for this, nice job! Have you submitted it to Laravel News?

**Comment by mikaelpopowicz** [comment]
Hi @aarondfrancis thank you for your kind words. We submitted the Github link to Laravel News.
